.flake8

```diff
@@ -1,19 +1,59 @@
 [flake8]
-# E203 skipped because it annoys black.
-# no pyproject.toml support https://gitlab.com/pycqa/flake8/-/issues/428
-show-source = true
-extend-ignore =
-  E203,
-  F401,
-max-line-length = 100
+
 builtins = _
+
+# Print the total number of errors:
+count = true
+
+# Don't even try to analyze these:
 extend-exclude =
-  # skip tool cache dirs
+  # No need to traverse egg info dir
```

Comment:
I see there in some attempt to order these in a particular fashion?  Can that be explained so future addition here respect the ordering?

Reply:
I guess. Since I'm synchronizing this across many projects, this is a kind of thing that would need to go into the comment above the setting and be copied over into many places. The order is alphabetical so that it produces the lowest possible amount of conflicts.

Reply:
IMO, if we expect future contributors to keep in alphabetical order it should be noted.  If not changed in this PR can you log an issue for follow-up so it's not forgotten?